algorithms/linfa-clustering/src/k_means/algorithm.rs

```diff
@@ -614,6 +605,33 @@ mod tests {
         y
     }
 
+    macro_rules! calc_inertia {
```

Comment:
any reason why this is a macro?

Reply:
Input types are long so I didn't want to write them all out just for a test helper

Reply:
sounds reasonable